File: GoLang/8 kyu/Multiplication table for number/main.go
package main

import "fmt"

func main() {
	fmt.Println(MultiTable(4))
}

func MultiTable(number int) string {
	res := ""
	for i := 1; i <= 10; i++ {
	  if i != 10 {
		res += fmt.Sprintf("%v * %v = %v\n", i, number, i*number)
	  } else {
		res += fmt.Sprintf("%v * %v = %v", i, number, i*number)
	  }
	  
	} 
	return res
  }

File: Python/8 kyu/Multiplication table for number/main.py
from audioop import mul


def multi_table(number):
    res = ""
    for i in range(1, 11):
        if i != 10:
            res += str(i) + " * " + str(number) + " = " + str(i*number) + "\n"
        else:
            res += str(i) + " * " + str(number) + " = " + str(i*number)
    return res

if __name__ == "__main__":
    print(multi_table(4))